sdcm/db_log_reader.py

```diff
@@ -29,6 +29,11 @@
 
 LOGGER = logging.getLogger(__name__)
 
+# we aren't going to process log line which are bigger than
+# this value, i.e. not event would be generated base on those,
+# but they would still be in the logs
+LOG_LINE_MAX_PROCESSING_SIZE = 1024 * 5
```

Comment:
Do you have examples of the big ones that cross this threshold?
Will we skip them even if they are errors?

What about the big stack trace we have for coredumps, will we miss them?

Reply:
Yes, if it's too big, we won't look for errors in it.

We were getting them from resharding print, that list all the sstables it processed in one line.

Coredumps info isn't coming via the log, we are calling coredumpctl and getting it directly, the parts in the log are multi lines, and don't get to those sizes.

One line backtraces can be long, but not 5K long

Reply:
I'm worried that the 5K is a random size you picked.
Is there a way to grep an existing log based on line size and see which messages falls into this criteria?

Reply:
I did this analysis on a 12h run, and the logs are compaction (reshape/reshard) summeries, and bootstrap that shows all the ranges.

none of those are something we match, and they don't print any errors in them.

regardless they are not gone, they would still be in the log.

the logs line we rely on, are not even close to that size.

what the worst thing that would happen ? we'll miss some silly error we gonna ignore, we'll miss a backtrace of a coredump we are catching, and someone would need to manually decode when he open a bug. (and then we tweak this number, I'm very doubtful it would ever happen)

Reply:
No, I'm afraid that the worst case we will miss critical log line that detect a corrupted sstable for example.

Reply:
when is the last time you've seen one of those critical logs line that is that big ?

I don't have any data to suggest there are long critical lines, is it worth doubling the number of cpus of the test machines  for ?